draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -942,10 +942,12 @@ MUST choose placeholder claim names accordingly.
 It is RECOMMENDED to use cryptographically salts with at least 128 bits
 of entropy as placeholder claim names.
 
-The order of elements in JSON-encoded objects is not relevant to applications,
-but the order may reveal information about the blinded claim name to the
-verifier. It is therefore RECOMMENDED to ensure that the order is shuffled or
-otherwise hidden (e.g., alphabetically ordered using the blinded claim names).
+The order of elements in JSON-encoded objects is not relevant to
+applications, but the order may reveal information about the blinded
+claim name to the verifier. It is therefore RECOMMENDED to ensure that
+the claims in the SD-JWT, II-Disclosures object, and HS-Disclosures JWT
+is shuffled or otherwise hidden, e.g., by alphabetically sorting using
```

Comment:
```suggestion
are shuffled or otherwise hidden, e.g., by alphabetically sorting using
```

Reply:
do we want to say it is RECOMMENDED to alphabetically sort..?

Reply:
all claims, or only blinded claim names? if the former, this sentence should be in another section, if the latter, needs to be clarified.

Reply:
Added clarifications!